korman/operators/op_mesh.py

```diff
@@ -0,0 +1,369 @@
+                                             description="Specifies whether the ladder has an upper entry",
+                                             default=True)
+    upper_entry_enabled = bpy.props.BoolProperty(name="Upper Entry Enabled",
+                                                 description="Specifies whether the ladder's upper entry is enabled by default at Age start",
+                                                 default=True)
+    has_lower_entry = bpy.props.BoolProperty(name="Has Lower Entry Point",
+                                             description="Specifies whether the ladder has a lower entry",
+                                             default=True)
+    lower_entry_enabled = bpy.props.BoolProperty(name="Lower Entry Enabled",
+                                                 description="Specifies whether the ladder's lower entry is enabled by default at Age start",
+                                                 default=True)
+
+    def draw(self, context):
+        layout = self.layout
+        space = bpy.context.space_data
+        if (not space.local_view):
+            box = layout.box()
+            box.label("Geometry:")
+
+            row = box.row()
+            if (self.ladder_height % 2 != 0):
+                row.alert = True
+            row.prop(self, "ladder_height")
+            row = box.row()
+            row.prop(self, "ladder_width")
+            row = box.row()
+            row.prop(self, "rung_height")
+
+            box = layout.box()
+            box.label("Template Guides:")
+            col = box.column()
+            col.prop(self, "gen_back_guide")
+            col.prop(self, "gen_ground_guides")
+            col.prop(self, "gen_rung_guides")
+            if self.gen_rung_guides:
+                col.separator()
+                col.prop(self, "rung_width_type", text="")
+
+            box = layout.box()
+            row = box.row()
+            col = row.column()
+            col.label("Upper Entry:")
+            col.row().prop(self, "has_upper_entry", text="Create")
+            row = col.row()
+            if not self.has_upper_entry:
+                row.enabled = False
+            row.prop(self, "upper_entry_enabled", text="Enabled")
+            col.separator()
+            col.label("Lower Entry:")
+            col.row().prop(self, "has_lower_entry", text="Create")
+            row = col.row()
+            if not self.has_lower_entry:
+                row.enabled = False
+            row.prop(self, "lower_entry_enabled", text="Enabled")
+
+        else:
+            row = layout.row()
+            row.label("Warning: Operator does not work in local view mode", icon="ERROR")
+
+    def execute(self, context):
+        if bpy.context.mode == "OBJECT":
+            create_ladder_objects(self, context)
+            return {"FINISHED"}
+        else:
+            self.report({'WARNING'}, "Ladder creation only valid in Object mode")
```

Comment:
I think it would be a good idea to overload the operator's `poll()` method to do this check as well

Reply:
What would polling for the context mode do in this case?

Reply:
If the poll fails, the operator should be hidden

Reply:
It already is, I think?  The Add menu in Edit Mode is a different menu and doesn't show this operator.  The other modes don't even have an Add menu.

I think you're correct, but I'm not sure it's necessary.  Unless I misunderstand the problem.